src/clang.rs

```diff
@@ -647,11 +647,17 @@ impl Type {
 
     /// Given that this type is a pointer type, return the type that it points
     /// to.
-    pub fn pointee_type(&self) -> Type {
-        unsafe {
-            Type {
-                x: clang_getPointeeType(self.x),
+    pub fn pointee_type(&self) -> Option<Type> {
+        match self.x.kind {
```

Comment:
can you use `self.kind()`? not a big deal in this case, but better to be consistent.